packages/tinacms/src/components/media/media-manager.tsx

```diff
@@ -361,7 +393,8 @@ export function MediaPicker({
               onChange={e => setSearchInput(e.target.value)}
             />
           </form>
-          <div>
+          <div className="button-container">
+            {renderCustomActionButtons(tabs[currentTab].customActions, customActionProps)}
```

Comment:
What if customActions is empty or undefined?

Reply:
Updated